js/src/rive.ts

```diff
@@ -0,0 +1,965 @@
+import Runtime from '../../wasm/publish/rive.js';
+
+// #region LoopEvent
+
+// Loop types. The index of the type is the value that comes from Wasm
+export const loopTypes: string[] = ['oneShot', 'loop', 'pingPong'];
+
+/// Loop events are returned through onloop callbacks
+interface LoopEvent {
+  animation: string;
+  name: string;
+  type: number;
+}
+
+// Creates a new LoopEvent
+export let createLoopEvent = (animation: string, loopValue: number): LoopEvent => {
```

Comment:
I don't think you want to mutate this variable. I would recommend to export it as a regular function, or as a const.
Also I think using string literal will provide a better devX than a number for the loopValue :
```typescript
function createLoopEvent(animation: string, loopType: LoopType)
```

Reply:
Cool; I'll look into that.